code/game/area/space_station_13_areas.dm

```diff
@@ -115,6 +115,9 @@ NOTE: there are two lists of areas in the end of this file: centcom and station
 	name = "AI Chamber"
 	icon_state = "ai_chamber"
 
+/area/ai_monitored/turret_protected/ai/Initialize()
+	ambientsounds -= 'sound/ambience/ambimalf.ogg' //This sound is terrible when you hear it constantly, so remove it from the core's sound pool.
```

Comment:
Surely this can't be the proper way to do this. Couldn't you just not add it to the ambientsounds list in the first place?

Reply:
I was going to override the list, but I figured if anyone wanted to add more AI ambient sounds later, having to do it in multiple spots seems dumb.

Reply:
Make a redefined proc adding this specific sound if you don't want to repeat code too much.
It's better than adding and removing.